src/ops/mod.rs

```diff
@@ -204,7 +206,8 @@ pub mod test {
                 .as_local()]
                     .borrow_mut()
                     .inner
-                    .on_input(src, u.into(), &self.nodes, &self.states);
+                    .on_input(src, u.into(), &self.nodes, &self.states)
```

Comment:
This one removes the semicolon from this line, and adds an unwrap() call right after.